main.html

```diff
@@ -0,0 +1,365 @@
+        </li>
+        <li class="navbar_list">
+          <a href="/main.html" class="logo"><h1>Fayob</h1> </a>
+        </li>
+        <li class="navbar_list">
+          <a href="#"
+            ><i id="search" class="fa fa-search fa-2x" aria-hidden="true"></i
+          ></a>
+        </li>
+      </ul>
+    </nav>
+    <nav class="desktop_navbar">
+      <div class="top_nav">
+        <ul class="top_nav_lists">
+          <li class="top_nav_list">
+            <a href="#" class="top_nav_list_a"
+              ><i class="fa fa-facebook"></i
+            ></a>
+          </li>
+          <li class="top_nav_list">
+            <a href="#" class="top_nav_list_a"><i class="fa fa-twitter"></i></a>
+          </li>
+          <li class="top_nav_list">
+            <a href="#" class="top_nav_list_a">English</a>
+          </li>
+          <li class="top_nav_list">
+            <a href="#" class="top_nav_list_a">My Page</a>
+          </li>
+          <li class="top_nav_list">
+            <a href="#" class="top_nav_list_a">Logout</a>
+          </li>
+        </ul>
+      </div>
+      <section class="desktop_nav">
+        <div>
+          <a href="/main.html" class="logo"><h1>Fayob</h1> </a>
+        </div>
+        <div>
+          <ul class="desktop_nav_lists">
+            <li class="desktop_nav_list"><a href="/about.html">About</a></li>
+            <li class="desktop_nav_list"><a href="#">Contact Us</a></li>
+            <li class="desktop_nav_list"><a href="#">Blog</a></li>
+            <li class="desktop_nav_list"><a href="#">Program</a></li>
+            <li class="desktop_nav_list"><a href="#">Help</a></li>
+            <li class="desktop_nav_list"><a href="#">Careers</a></li>
+          </ul>
+        </div>
+      </section>
+    </nav>
+
+    <!-- Side Section -->
+    <aside class="search_box">
+      <input type="text" placeholder="Search for your favorite course" />
+      <button type="button">Search</button>
+    </aside>
+    <aside class="menu_options">
+      <a href="#"
+        ><i class="fa fa-close fa-2x" id="close_bar" aria-hidden="true"></i
+      ></a>
+      <ul>
+        <li><a href="/about.html">About</a></li>
+        <li><a href="#">Contact Us</a></li>
+        <li><a href="#">Blog</a></li>
+        <li><a href="#">Programmes</a></li>
+        <li><a href="#">Help</a></li>
+        <li><a href="#">Careers</a></li>
+      </ul>
+    </aside>
+
+    <!-- Main Section -->
+    <main>
+      <header class="welcome_header">
+        <div class="welcome_header_title">
+          <h4>"Hello! Sharing world"</h4>
+          <h2>Welcome to Fayob online learning platform</h2>
+        </div>
+
+        <p class="header_paragraph">
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Similique
+          corrupti tenetur aspernatur repellat ex assumenda omnis, ducimus
+          provident, accusamus deserunt culpa soluta a, illo laborum voluptas
+          eaque dolores sint minus.
+        </p>
+      </header>
```

Comment:
- [ ] Kindly align the text to the left and reduce the width of `header_paragraph` like the one of Behance.